Tests/Runner/DataLoaderMetadataTest.php

```diff
@@ -346,6 +359,7 @@ public function testExecutorManifestMetadata()
 
     public function testExecutorManifestMetadataCombined()
     {
+        self::markTestSkipped('TODO');
```

Comment:
Kdy se to bude `TODO`?

Reply:
Až releasneš tohle https://github.com/keboola/output-mapping/pull/94 :)

Reply:
Tak jsem si to releasnul sám, to bych se taky jinak nemusel dočkat :)

Reply:
@joker806 fixed